packages/expo-notifications/src/getExpoPushTokenAsync.ts

```diff
@@ -0,0 +1,182 @@
+import { Platform, CodedError } from '@unimodules/core';
+import * as Application from 'expo-application';
+import Constants from 'expo-constants';
+
+import InstallationIdProvider from './InstallationIdProvider';
+import getDevicePushTokenAsync, { DevicePushToken } from './getDevicePushTokenAsync';
+
+const productionBaseUrl = 'https://exp.host/--/api/v2/';
+
+export interface ExpoPushToken {
+  type: 'expo';
+  data: string;
+}
+
+interface Options {
+  // Endpoint URL override
+  baseUrl?: string;
+
+  // Request URL override
+  url?: string;
+
+  // Request body overrides
+  type?: string;
+  appId?: string;
+  deviceId?: string;
+  development?: boolean;
+  experienceId?: string;
+  devicePushToken?: DevicePushToken;
+}
+
+export default async function getExpoPushTokenAsync(options: Options = {}): Promise<ExpoPushToken> {
+  const devicePushToken = options.devicePushToken || (await getDevicePushTokenAsync());
+
+  const deviceId = options.deviceId || (await getDeviceIdAsync());
+
+  const experienceId =
+    options.experienceId ||
+    (Constants.manifest && `@${Constants.manifest.owner}/${Constants.manifest.slug}`);
```

Comment:
```suggestion
    (Constants.manifest && Constants.manifest.id);
```
we actually want to move away from the `@owner/slug` construction of experienceId so let's not add another dependency on it here 😅